docs/contributing/CHANGELOG_TEMPLATE.md

```diff
@@ -4,26 +4,21 @@
 
 💥 Breaking changes:
 
-- Pull Request Title goes here
+Heading (h4) to summarise change goes here
 
-  Description goes here (optional)
+Description goes here
 
-  To migrate you need to change: X
-
-  ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
+This change was introduced in [pull request #xxxx: PULL-REQUEST-TITLE](LINK-TO-PULL-REQUEST). (optional) Thanks to [@GITHUB-NAME](LINK-TO-CONTRIBUTOR-GITHUB-PAGE) for reporting/contributing this issue.
```

Comment:
Can we add something around the instances of link text, so that there's no actual link and users can see the text "(LINK-TO-PULL-REQUEST)" and "(LINK-TO-CONTRIBUTOR-GITHUB-PAGE)"?

Same question for line 19. :)

Reply:
@EoinShaughnessy We could wrap it in backticks, so it interprets it as code and doesn't force a link?

```suggestion
This change was introduced in `[pull request #xxxx: PULL-REQUEST-TITLE](LINK-TO-PULL-REQUEST)`. (optional) Thanks to `[@GITHUB-NAME](LINK-TO-CONTRIBUTOR-GITHUB-PAGE)` for reporting/contributing this issue.
```

Reply:
I'm not clear what the "(optional)" bit here is referring to - do we mean the contributor shout-out is optional? I wonder if we can make it clearer which bit it refers to.

Reply:
That sounds good, thanks @EoinShaughnessy !